indico/modules/events/templates/display/conference/base.html

```diff
@@ -5,19 +5,23 @@
 
 {%- macro _format_event_date(event) -%}
     {% set start_dt = event.start_dt_display.astimezone(event.display_tzinfo) %}
+    {% set start_dt_iso = event.start_dt_display.isoformat() %}
```

Comment:
is there any benefit in using UTC here instead of keeping the timezone? if not, just use `start_dt.isoformat()`

Reply:
Changed